www/source/blog/2017-11-16-angular-seed.html.md

```diff
@@ -0,0 +1,117 @@
+---
+title: NodeJS Scaffolding with Angular Seed!
+date: 2017-11-16
+author: Nell Shamrell-Harrington
+tags: Scaffolding, Node, Angular
+category: community
+classes: body-article
+---
+
+Hello my fellow Habicats!
+
+Do you have legacy Angular projects that use [angular-seed](https://github.com/mgechev/angular-seed)? Now you can package them with Habitat using the Node Scaffolding! Let's give it a go!
+
+## Pre-requisites
+* A Github Account
+* NodeJS installed on your workstation
```

Comment:
Any minimum version requirement (or maybe a note to refer to the README or package.json)? The angular-seed readme seems to `strongly recommend node >=v6.5.0 and npm >=3.10.3 `.

Reply:
Will add that in!